ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/port_channel_interfaces.py

```diff
@@ -75,6 +75,11 @@ def port_channel_interfaces(self) -> list | None:
                 }
                 port_channel_interface["lacp_id"] = generate_lacp_id(short_esi)
 
+            if self.shared_utils.underlay_isis:
```

Comment:
Currently we only support L2 port-channels, so we should *not* introduce ISIS config here.

Reply:
Removed!